.github/workflows/e2e.yml

```diff
@@ -0,0 +1,40 @@
+name: E2E
+
+on:
+  pull_request:
+    types: [ready_for_review]
```

Comment:
Will this also re-run the tests when you push new changes to a PR that is already in ready for review?

Reply:
I'm afraid not at the moment. I could probably add on `push` as type and than a conditional that checks if the PR was already marked as `ready_for_review`.